docs/reference/logging/fluentd.md

```diff
@@ -34,7 +34,8 @@ Some options are supported by specifying `--log-opt` as many times as needed:
 
  - `fluentd-address`: specify `host:port` to connect `localhost:24224`
  - `tag`: specify tag for fluentd message, which interpret some markup, ex `{{.ID}}`, `{{.FullID}}` or `{{.Name}}` `docker.{{.ID}}`
-
+ - `fail-on-startup-error`: true/false; Should the logging driver fail container startup in case of connect error during startup. Default: true (backwards compatible)
```

Comment:
These options should be detailed in `## Options` too (just below : https://github.com/docker/docker/pull/18041/files#diff-9a6be46aa59594b9640ede14e4bcc8acR59) and in more detail (or these a little more summarized — e.g. default value, etc.).